api/admin/controller/__init__.py

```diff
@@ -796,6 +797,7 @@ def custom_lists(self):
                         entry_count=list.size,
                         auto_update=list.auto_update_enabled,
                         auto_update_query=list.auto_update_query,
+                        auto_update_facets=list.auto_update_facets,
```

Comment:
Lines 782 to 800  and  804 to 824 are virtually identical.  It seems that these could be refactored in a single method to reduce code duplication.

Reply:
I was having similar anxiety while making this change. Will fix.